polymetis/polymetis/install.sh

```diff
@@ -29,7 +29,6 @@ if [ -z "$BUILD_FRANKA" ]; then
 fi
 
 if [ "$BUILD_FRANKA" == "ON" ]; then
-    if [ -d "$BUILD_PATH" ]; then rm -r $BUILD_PATH; fi
```

Comment:
What's the logic here?

Reply:
The install script would delete the build directory of libfranka and rebuild everything from scratch every time I run it with `BUILD_FRANKA=ON`, which leads to unnecessarily long debugging times. (`BUILD_FRANKA` needs to be `ON` in order to rebuild hardware clients)